forwards_paper.tex

```diff
@@ -238,29 +238,25 @@ \subsection*{Simulation benchmarks}
 neutral mutations (instead of the pedigree), run times increase dramatically with increasing region size ($4Nr = 4Nu$ for
 these simulations).  With \fwdpy{}, simulations with $N=5 \times 10^4$ timed out for region sizes larger than $10^3$.
 With \simupop{}, we were only able to run simulations with $N=10^3$ and region sizes up to $10^4$.  Pedigree tracking
 dramatically improves run times for both \fwdpy{} and \simupop{} (right column of Figure~\ref{fig:runtimes_selection}).  For parameters that ran to completion when tracking neutral mutations and when tracking
-the pedigree, the relative speedup due to pedigree tracking is up to $\approx 50$ fold (Figure~\ref{fig:relative_speedup_selection}).  For \fwdpy{}, pedigree tracking reduces runtimes more for larger population sizes
-(Figure~\ref{fig:relative_speedup_selection}).  We could not make such an observation with \simupop{}, which tended to
-run out of memory on our system.  Note the log scale of the x axis in Figure~\ref{fig:runtimes_selection}, which
+the pedigree, the relative speedup due to pedigree tracking is up to $\approx 50$ fold
+(Figure~\ref{fig:relative_speedup_selection}).  For \fwdpy{}, pedigree tracking reduces runtimes more for larger
+population sizes and we observe increasing relative speedups as $4Nr$ increases for a given $N$
+(Figure~\ref{fig:relative_speedup_selection}).  We saw a similar trend with $4Nr$ for \simupop{}, but the effect of $N$
+could not be explored because the simulations tended to run out of available memory.
+  Note the log scale of the x axis in Figure~\ref{fig:runtimes_selection}, which
 partially obscures an important observation that the run times appear approximately linear in region size when pedigree
-tracking.  The reason for this behavior is that the simulations are generating a constant number of new edges on average
-each generation, and the number of edges is a function of $\rho$ in expectation.
-
-We also performed a more limited set of simulations without natural selection.  The total run times are shown in
-Figure~\ref{sfig:rawspeed_nosel} and show the same qualitative behavior as simulations with selection
-(Figure~\ref{fig:runtimes_selection}).  The relative improvement due to pedigree tracking is again substantial in these
-simulations (Figure~\ref{sfig:speedup_nosel}).  In fact, we see more of a benefit to pedigree tracking here with \fwdpy{}
-than we did with selection (Figure~\ref{fig:relative_speedup_selection}).  The reason is that the fitness function exits
-close to instantly in simulations without selection that are based on \fwdpp{}, meaning that \fwdpy{} is doing little
-more than generating random numbers and book-keeping in Figures~\ref{sfig:rawspeed_nosel}
-and~\ref{sfig:speedup_nosel}.
+tracking.  The reason for this behavior is that the simulations are generating a constant number of new edges on
+average
+each generation as the number of new recombination breakpoints is Poisson distributed with expectation $\rho$.
```

Comment:
perhaps clarify: "as the total number of new reomcombination breakpoints in the population
is Poisson distributed with expectation $\rho$"